Description: Disable commit hash and date
 Disabling commit hash and compiled date as they are not needed in the
 Debian package. These are shown when you invoke `micro --version`.
Author: Jongmin Kim
Last-Update: 25/12/2019

--- a/cmd/micro/micro.go
+++ b/cmd/micro/micro.go
@@ -312,8 +312,8 @@
 	if *flagVersion {
 		// If -version was passed
 		fmt.Println("Version:", Version)
-		fmt.Println("Commit hash:", CommitHash)
-		fmt.Println("Compiled on", CompileDate)
+		//fmt.Println("Commit hash:", CommitHash)
+		//fmt.Println("Compiled on", CompileDate)
 		os.Exit(0)
 	}
 
